src/sentry/sentry_metrics/indexer/cache.py

```diff
@@ -10,10 +11,16 @@
 logger = logging.getLogger(__name__)
 
 
+class PartitionKey(Enum):
+    CLOUD_SPANNER = "cs"
+    POSTGRES = "pg"
```

Comment:
I'd prefer if the cache was independent of the storage layer (cloudspanner or postgres). Can we just pass a string here instead?

Reply:
ok updated